Hazelcast.Net/Hazelcast.IO.Serialization/HazelcastJsonValue.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Hazelcast.Util;
+using System;
+
+namespace Hazelcast.IO.Serialization
+{
+    /// <summary>
+    ///  Class for serializing/deserializing Hazelcast JSON types
```

Comment:
API doc need to be improved. Please use the doc on Java API and document other methods:
`ToString` `FromString`

Reply:
Updated